test.js

```diff
@@ -296,7 +297,7 @@ test('enqueue - skip when client is disabled', async t => {
 test('flush - don\'t fail when queue is empty', async t => {
   const client = createClient()
 
-  await t.notThrows(client.flush())
+  await t.notThrowsAsync(() => client.flush())
```

Comment:
For anything that returns a promise, we have to use `t.throwsAsync` or `t.notThrowsAsync` now